07.ls_object/separator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Separator
+  def initialize(paths)
+    @paths = paths
+  end
+
+  def fetch_file(need_reverse_order = nil)
```

Comment:
true/falseを想定しているなら、

```suggestion
  def fetch_file(need_reverse_order = false)
```

の方がわかりやすいと思います。

あと「〜する必要がある」はrequiredもよく使われます。
`reverse_required`ぐらいが短くて良いかもしれません。

Reply:
`false`を使うようにします。

`need_reverse_order`は`reverse_required`に変更します。